runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -217,33 +219,52 @@ public void processFlash(final Graphics2D graphics)
 			return;
 		}
 
-		if (Instant.now().minusMillis(MINIMUM_FLASH_DURATION_MILLIS).isAfter(flashStart))
+		final int flash_notification_duration_config_val = runeLiteConfig.flashNotificationDuration();
+		final int flash_duration_millis;
+
+
+		if (flash_notification_duration_config_val <= 0)
+		{
+			flash_duration_millis = 0;
+		}
+		else if (flash_notification_duration_config_val < MINIMUM_FLASH_DURATION_MILLIS)
+		{
+			// If less than minimum duration but not set to 0 then set to minimum.
+			flash_duration_millis = MINIMUM_FLASH_DURATION_MILLIS;
+		}
+		else
+		{
+			flash_duration_millis = flash_notification_duration_config_val;
+		}
```

Comment:
This is just `else { flash_duration_millis = Math.min(MINIMUM_FLASH_DURATION_MILLIS, flash_notification_duration_config_val); }`
```suggestion
		else
		{
			flash_duration_millis = Math.min(MINIMUM_FLASH_DURATION_MILLIS, flash_notification_duration_config_val);
		}
```